vars/sectionTriggerFluxDeployment.groovy

```diff
@@ -0,0 +1,39 @@
+#!groovy
+import uk.gov.hmcts.contino.AppPipelineConfig
+import uk.gov.hmcts.contino.PipelineCallbacksRunner
+import uk.gov.hmcts.contino.PipelineType
+import uk.gov.hmcts.contino.DockerImage
+import uk.gov.hmcts.contino.ProjectBranch
+import uk.gov.hmcts.contino.azure.Acr
+
+/*
+ * The image retaging in ACR is used as a trigger for Weaveworks Flux
+ * to (re-)deploy the image
+ *
+ * Any image re-tagged following the pattern displayed below is
+ * eventually deployed in Flux AKS cluster:
+ *
+ * e.g.: <my-app-image>:latest-<random-hash>
+ */
+
+def call(params) {
+
+  PipelineCallbacksRunner pcr = params.pipelineCallbacksRunner
+  AppPipelineConfig config = params.appPipelineConfig
+  PipelineType pipelineType = params.pipelineType
+
+  def subscription = params.subscription
+  def product = params.product
+  def component = params.component
+
+  stage('AAT Flux deployment') {
+    pcr.callAround('fluxdeployment') {
+      withAksClient(subscription) {
```

Comment:
Currently the docker image flag isn't on by default, I think this needs to have the docker image feature flag around this stage?

Reply:
Is `pcr.callAround('fluxdeployment')` the feature flag call itself ? I though this was used for metrics or something. Not sure what this was made for so I followed the convention.

Reply:
no its not, it used to be called pl.dockerBuild, it's been renamed to something else now but the docker build stage should have the flag around it

Reply:
you need both the pcr and the flag